cmd/tailscale/main.go

```diff
@@ -858,8 +1059,108 @@ func (a *App) processUIEvents(w *app.Window, events []UIEvent, act jni.Object, s
 		case SearchEvent:
 			state.query = strings.ToLower(e.Query)
 			a.updateState(act, state)
+		case FileSendEvent:
+			a.sendFiles(e, files)
+		}
+	}
+}
+
+func (a *App) sendFiles(e FileSendEvent, files []File) {
+	go func() {
```

Comment:
I'd prefer longish-lived goroutines to have names. Makes reading debug stacks easier.

`go a.sendFilesAsync(...)` ? or `Routine`, etc.

Reply:
Routine is good. Thanks.